packages/react/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'clsx';
+
+export type BadgeColor =
+  | 'blue-light'
+  | 'green'
+  | 'green-light'
+  | 'red-light'
+  | 'orange-light'
+  | 'yellow';
```

Comment:
Dette holder sikkert med farger, men kunne jo lagt til en `'custom'` som ikke gir noen farge i `badgeVariations`, men krever at du legger det inn i `className`?

Reply:
Kan kanskje legge til støtte for det, skal se på det idag

Reply:
hvis du ikke sender inn color-prop så er defautl grønn, og kan overstyre via `className` propen